src/lib/y2firstboot/clients/configuration_management.rb

```diff
@@ -1,4 +1,4 @@
-# encoding: utf-8
+# frozen_string_literal: true
```

Comment:
those frozen strings are quire dangerous as it can cause exceptions when it tries to modify string literal. Is all test properly covered?

Reply:
Yep, you are right